test/functional/apps/context/_date_nanos.ts

```diff
@@ -28,6 +28,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await kibanaServer.uiSettings.update({
         'context:defaultSize': `${TEST_DEFAULT_CONTEXT_SIZE}`,
         'context:step': `${TEST_STEP_SIZE}`,
+        'doc_table:legacy': true,
```

Comment:
Note there are cases we need to clean up later on, think it's fine, since it's not the doc table that's being tested in these cases

Reply:
Maybe add an issue to track cleanup of these tests later on?